Makefile

```
@@ -0,0 +1,2 @@
+rb:
```

```suggestion
build-rust-bindings:
```

no need for  people having to guess what this means :)